src/expr/kind_template.cpp

```diff
@@ -101,22 +90,11 @@ TheoryId kindToTheoryId(::CVC4::Kind k) {
 TheoryId typeConstantToTheoryId(::CVC4::TypeConstant typeConstant) {
   switch(typeConstant) {
 ${type_constant_to_theory_id}
-#line 105 "${template}"
-  case LAST_TYPE:
-    break;
+#line 94 "${template}"
+case LAST_TYPE: break;
```

Comment:
Wrong indentation.

Reply:
Interesting. Clang-format screwed that up. I formatted the whole function now.